libs/core/src/validator/validatorHelper.ts

```diff
@@ -0,0 +1,22 @@
+export class ValidatorHelper {
```

Comment:
Would it be worth writing a test for this class?

Reply:
@ofuochi can you explain this?